src/lib/passport/test/passport/callbacks.js

```diff
@@ -1,107 +1,146 @@
-const Backbone = require('backbone');
-const rewire = require('rewire');
-const sinon = require('sinon');
-const cbs = rewire('../../lib/passport/callbacks');
+const Backbone = require("backbone")
+const rewire = require("rewire")
+const sinon = require("sinon")
+const cbs = rewire("../../lib/passport/callbacks")
 
-describe('passport callbacks', function() {
+/* eslint-disable jest/expect-expect, jest/no-done-callback */
+// TODO: Refactor artsy-passport code and take care of these warnings!
```

Comment:
It seems out of scope to refactor these tests in this PR.

Reply:
Totally! don't worry about it.